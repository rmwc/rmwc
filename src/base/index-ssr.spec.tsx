import React from 'react';
import { renderToString as mount } from 'react-dom/server';
import { Portal } from './';

describe('Base SSR', () => {
  it('renders', () => {
    const portal = mount(<Portal />);
    expect(portal).toMatchSnapshot();
  });
});
