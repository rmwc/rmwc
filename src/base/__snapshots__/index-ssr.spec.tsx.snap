// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base SSR renders 1`] = `"<div id=\\"rmwcPortal\\" data-reactroot=\\"\\"></div>"`;
